source/lambda/documentprocessor/lambda_function.py

```diff
@@ -62,22 +59,22 @@ def processRequest(request):
 
     print("Input Object: {}/{}".format(bucketName, objectName))
 
-    ext = FileHelper.getFileExtension(objectName.lower())
-    is_bad_mime_type = is_invalid_mime_type(request)
-
     client = AwsHelper().getClient('sqs')
+
+    file_type = get_mime_type(request)
+
     # If not expected extension, change status to FAILED and exit
-    if(is_bad_mime_type or (ext and ext not in ["jpg", "jpeg", "png", "pdf"])):
+    if(file_type and file_type not in ['application/pdf', 'image/png', 'image/jpeg']):
```

Comment:
I think the change not doesnt check if file type is None now.

Reply:
Updated